__tests__/e2e/cms/contentful.test.ts

```diff
@@ -0,0 +1,21 @@
+import { config as configDotenv } from 'dotenv';
+configDotenv();
+
+import { Contentful } from '../../../src/cms/contentful';
+
+describe('contentful', () => {
+  describe('should not fail', () => {
```

Comment:
Should we add a test for a failing case? e.g. trying to fetch a non-existing entry

Reply:
I assume the CF API has a well declared return type for such case. Although, it is defo a good idea to check if we also return the expected value for that.